doctoshotgun.py

```diff
@@ -547,54 +547,74 @@ def try_to_book_place(self, profile_id, motive_id, practice_id, agenda_ids, vac_
 
         return self.page.doc['confirmed']
 
+class Vaccine:
+    def __init__(self, key, motive):
+        self.key = key
+        self.motive = motive
+class Pfizer(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
+class Moderna(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
+class Janssen(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
+class AstraZeneca(Vaccine):
+    def __init__(self, key, motive):
+        super().__init__(key, motive)
 
 class DoctolibDE(Doctolib):
     BASEURL = 'https://www.doctolib.de'
-    KEY_PFIZER = '6768'
-    KEY_PFIZER_SECOND = '6769'
-    KEY_PFIZER_THIRD = None
-    KEY_MODERNA = '6936'
-    KEY_MODERNA_SECOND = '6937'
-    KEY_MODERNA_THIRD = None
-    KEY_JANSSEN = '7978'
-    KEY_ASTRAZENECA = '7109'
-    KEY_ASTRAZENECA_SECOND = '7110'
+    VACCINES = {
+        'PFIZER' : Pfizer('6768', 'Pfizer'),
+        'PFIZER_SECOND' : Pfizer('6769', 'Zweit.*Pfizer|Pfizer.*Zweit'), 
+        'PFIZER_THIRD' : Pfizer(None, 'Dritt.*Pfizer|Pfizer.*Dritt'),
+        'MODERNA' : Moderna('6936', 'Moderna'),
+        'MODERNA_SECOND' : Moderna('6937', 'Zweit.*Moderna|Moderna.*Zweit'),
+        'MODERNA_THIRD' : Moderna(None, 'Dritt.*Moderna|Moderna.*Dritt'),
+        'JANSSEN' : Janssen('7978','Janssen'),
+        'ASTRAZENECA' : AstraZeneca('7109','AstraZeneca'),
+        'ASTRAZENECA_SECOND' : AstraZeneca('7110', 'Zweit.*AstraZeneca|AstraZeneca.*Zweit')
+    }
     vaccine_motives = {
-        KEY_PFIZER: 'Pfizer',
-        KEY_PFIZER_SECOND: 'Zweit.*Pfizer|Pfizer.*Zweit',
-        KEY_PFIZER_THIRD: 'Dritt.*Pfizer|Pfizer.*Dritt',
-        KEY_MODERNA: 'Moderna',
-        KEY_MODERNA_SECOND: 'Zweit.*Moderna|Moderna.*Zweit',
-        KEY_MODERNA_THIRD: 'Dritt.*Moderna|Moderna.*Dritt',
-        KEY_JANSSEN: 'Janssen',
-        KEY_ASTRAZENECA: 'AstraZeneca',
-        KEY_ASTRAZENECA_SECOND: 'Zweit.*AstraZeneca|AstraZeneca.*Zweit',
+        VACCINES['PFIZER'].key : VACCINES['PFIZER'].motive,
+        VACCINES['PFIZER_SECOND'].key :  VACCINES['PFIZER_SECOND'].motive,
+        VACCINES['PFIZER_THIRD'].key : VACCINES['PFIZER_THIRD'].motive,
+        VACCINES['MODERNA'].key : VACCINES['MODERNA'].motive,
+        VACCINES['MODERNA_SECOND'].key : VACCINES['MODERNA_SECOND'].motive,
+        VACCINES['MODERNA_THIRD'].key : VACCINES['MODERNA_THIRD'].motive,
+        VACCINES['JANSSEN'].key : VACCINES['JANSSEN'].motive,
+        VACCINES['ASTRAZENECA'].key: VACCINES['ASTRAZENECA'].motive,
+        VACCINES['ASTRAZENECA_SECOND'].key: VACCINES['ASTRAZENECA_SECOND'].motive
```

Comment:
I think, instead of re-defining all key/motives, it is better to add a lookup method like `get_vaccine_motive(vaccine_key)` which iterates on the `VACCINES` dictionary.

Reply:
added function to VaccineBatch class